nixos/modules/programs/vim.nix

```diff
@@ -12,12 +12,25 @@ in {
       description = ''
         When enabled, installs vim and configures vim to be the default editor
         using the EDITOR environment variable.
+        '';
+    };
+
+    vimrc = mkOption {
+      type = types.lines;
+      default = "";
+      description = ''
+        Global vimrc configuration
       '';
     };
   };
 
   config = mkIf cfg.defaultEditor {
```

Comment:
this means, to set `/etc/vimrc`, vim has to be the default editor?

Reply:
Good point, will fix that.